### PR TITLE
Pipelines 5988: Update override for git resolver

### PR DESCRIPTION
**Pull request by nebelsprachen**
Version(s):

* Cherrypick to `pipelines-docs-1.14` `pipelines-docs-1.15` `pipelines-docs-1.16`

Issue:

* https://issues.redhat.com/browse/RHDEVDOCS-5988

Link to docs preview: https://78382--ocpdocs-pr.netlify.app/openshift-pipelines/latest/create/remote-pipelines-tasks-resolvers.html 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by ppitonak — CHANGES_REQUESTED** [review]

File: modules/op-resolver-git-scm.adoc [hunk 1]
```diff
@@ -68,7 +68,7 @@ The following example pipeline run references a remote pipeline from a Git repos
 
 [source,yaml]
 ----
-apiVersion: tekton.dev/v1
+apiVersion: tekton.dev/v1beta1
```
[the remark this answers is not in the document]
Comment (ppitonak) [reply modules/op-resolver-git-scm.adoc:71]: we should use v1 everywher

File: modules/op-resolver-git-anon.adoc [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        value: https://github.com/tektoncd/catalog.git
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: pipeline/simple/0.1/simple.yaml
+      - name: sample-pipeline-parameter
+        value: test
+  params:
+    - name: name
+      value: "testPipelineRun"
```
Comment (ppitonak) [inline modules/op-resolver-git-anon.adoc:76]:
```suggestion
      value: "testPipelineRun"
    - name: sample-pipeline-parameter
      value: test
```

File: modules/op-resolver-git-anon.adoc [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    resolver: git
+    params:
+      - name: url
+        value: https://github.com/tektoncd/catalog.git
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: pipeline/simple/0.1/simple.yaml
+      - name: sample-pipeline-parameter
+        value: test
```
[the remark this answers is not in the document]
Comment (ppitonak) [reply modules/op-resolver-git-anon.adoc:73]: see the suggestion below

File: modules/op-resolver-git-anon.adoc [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
```
Comment (ppitonak) [inline modules/op-resolver-git-anon.adoc:73]:
```suggestion
```

File: modules/op-resolver-git-anon.adoc [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+        params:
+          - name: url
+            value: https://github.com/tektoncd/catalog.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: task/git-clone/0.6/git-clone.yaml
+    params:        
+    - name: sample-task-parameter
+      value: test
```
Comment (ppitonak) [inline modules/op-resolver-git-anon.adoc:103]:
```suggestion
      params:        
      - name: sample-task-parameter
        value: test
```

File: modules/op-resolver-git-override-scm.adoc [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+      value: task/git-clone/0.6/git-clone.yaml
+    - name: token
+      value: my-secret-token
+    - name: tokenKey
+      value: token
+    - name: scmType
+      value: github
+    - name: serverURL
+      value: https://ghe.mycompany.com
+
```
Comment (ppitonak) [inline modules/op-resolver-git-override-scm.adoc:39]:
```suggestion
    params:
    - name: org
      value: tektoncd
    - name: repo
      value: catalog
    - name: revision
      value: main
    - name: pathInRepo
      value: task/git-clone/0.6/git-clone.yaml
    - name: token
      value: my-secret-token
    - name: tokenKey
      value: token
    - name: scmType
      value: github
    - name: serverURL
      value: https://ghe.mycompany.com
```

**Review by ppitonak — CHANGES_REQUESTED** [review]

**Review by vdemeester — APPROVED** [review]

File: modules/op-resolver-git-override-scm.adoc [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+| The key name for the token.
+| token
+
+| scmType
+| The type of SCM (Source Control Management) system.
+| github
+
+| serverURL
+| The URL of the server hosting the repository.
+| https://ghe.mycompany.com
```
Comment (adellape) [inline modules/op-resolver-git-override-scm.adoc:74]: This needs some passthrough to make it non-clickable (like the others in earlier tables):
```suggestion
| `+https://ghe.mycompany.com+`
```

File: modules/op-resolver-git-override-scm.adoc [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+      value: https://ghe.mycompany.com
+----
++
+.Supported parameters to override the Git resolver
+|===
+| Parameter | Description | Example value
+
+| org
+| The organization for the repository.
+| tektoncd
```
Comment (adellape) [inline modules/op-resolver-git-override-scm.adoc:46]: Overall, this table should be marked-up/formatted (for file names, field/value names, etc.) to better match the earlier tables, too. For example in this first row (though, apply uniformly throughout the table):
```suggestion
.Supported parameters to override the Git resolver
|===
| Parameter | Description | Example value

| `org`
| The organization for the repository.
| `tektoncd`
```

**Comment by adellape** [comment]
Also reminder to squash down to 1 commit before requesting merge. Thank you!

**Review by ppitonak — APPROVED** [review]

File: modules/op-resolver-git-override-scm.adoc [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+| The key name for the token.
+| token
+
+| scmType
+| The type of SCM (Source Control Management) system.
+| github
+
+| serverURL
+| The URL of the server hosting the repository.
+| `+https://ghe.mycompany.com+`
```
Comment (adellape) [inline modules/op-resolver-git-override-scm.adoc:73]: I meant for the whole table to get the marked-up treatment (where appropriate), but not going to block merge on this. But consider for a follow-up.

**Review by adellape — APPROVED** [review]

**Pull request merged**